overlord/snapstate/handlers_link_test.go

```diff
@@ -1504,6 +1504,45 @@ func (s *linkSnapSuite) TestDoUndoLinkSnapRestoresRefreshInhibitedTime(c *C) {
 	c.Check(snapst.RefreshInhibitedTime.Equal(instant), Equals, true)
 }
 
+func (s *linkSnapSuite) TestDoUndoLinkSnapRestoresLastRefreshTime(c *C) {
+	s.state.Lock()
+	defer s.state.Unlock()
+
+	now, err := time.Parse(time.RFC3339, "2021-06-10T10:00:00Z")
+	c.Assert(err, IsNil)
```

Comment:
I'm confused don't we need to use MockTimeNow with this?  is there some global mocking I'm missing or some mocking that is not restored?

Reply:
It was implicitly relying on not mocking timeNow, so would fail if undo didn't do the right thing (real current time would be set), but I see how this is confusing, so made it more explicit now.